osf/management/commands/create_notifications.py

```diff
@@ -0,0 +1,27 @@
+from django.core.management.base import BaseCommand
```

Comment:
nit: I would change the change the name of the file to indicate it's for testing only. People might not read the help text fully.